dev-requirements.txt

```diff
@@ -14,3 +14,4 @@ plotly>=2.0.8
 requests[security]
 flake8
 pylint==1.9.2
+Cerberus==1.2
```

Comment:
You also need to add it in setup.py `install_requires` since it is directly used by the code.